alacritty/src/renderer/mod.rs

```diff
@@ -417,6 +417,7 @@ struct InstanceData {
     bg_g: f32,
     bg_b: f32,
     bg_a: f32,
+    colored: i32,
```

Comment:
Can we use something lower on memory for boolean flag? I feel like sending i32 is a bit costly, right? However I remember that it wasn't the case on macOS for f16 types somehow in the past.

Reply:
I'm quite new with shaders actually. Also was unhappy with that but AFAIK, the choice is between a bool and  int in the glsl but passing booleans this way is apparently forbidden (probably meant to be used with uniform only).

But it looks like changing this field to u8 and keeping `int` in the shader works fine, so I'll switch to that for now.

Reply:
Yeah, bool is just u8, I was not sure about sending a bit more data.